airflow/config_templates/config.yml

```
@@ -1334,6 +1334,13 @@ hive:
 webserver:
   description: ~
   options:
+    config_file:
+      description: |
+        Path of webserver config file used for configuring the webserver parameters
+      version_added: 2.6.0
```

This will be 2.7.0 now.

```suggestion
      version_added: 2.7.0
```

---

Thank you. Made the changes